quickannotator/api/v1/tile/__init__.py

```diff
@@ -110,11 +110,11 @@ def get(self, args):
         """     get all Tiles within a bounding box
         """
         
-        tilespace = get_tilespace(args['annotation_class_id'], args['image_id'], in_work_mag=False)
-        tile_ids_in_bbox = tilespace.get_tile_ids_within_bbox(args['x1'], args['y1'], args['x2'], args['y2'])
+        tilespace = get_tilespace(image_id=args['image_id'], annotation_class_id=args['annotation_class_id'], in_work_mag=False)
+        tile_ids_in_bbox = tilespace.get_tile_ids_within_bbox((args['x1'], args['y1'], args['x2'], args['y2']))
         tile_ids_in_mask, _, _ = get_tile_ids_intersecting_mask(args['image_id'], args['annotation_class_id'], mask_dilation=1)
         ids = set(tile_ids_in_bbox) & set(tile_ids_in_mask)
-
+        tilespace.get_all_tile_ids_for_image()
```

Comment:
does this line of code do anything? it doesn't seem to have its result stored?

Reply:
Nice catch - this must have been a typo. I've removed this line of code.